### PR TITLE
Rename WRITE A POST to CREATE POST

### PR DESCRIPTION
**Pull request by lisasy**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Update the naming of the create post entrypoint from "Write a post" to "Create post" optimizing for a few things:
1. this makes clear that this composer isn't going to be limited to merely long-form essay format.
2. the new language is more concise

### Before
Screen Shot 2021-05-25 at 8 37 00 AM

### Proposed
Screen Shot 2021-05-25 at 8 37 19 AM

### UI accessibility concerns?
None

## Added tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests
- [x] I modified tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [x] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -1415,7 +1415,6 @@
     t.index ["reset_password_token"], name: "index_users_on_reset_password_token", unique: true
     t.index ["twitter_username"], name: "index_users_on_twitter_username", unique: true
     t.index ["username"], name: "index_users_on_username", unique: true
-    t.check_constraint "username IS NOT NULL", name: "users_username_not_null"
```
Comment (Ridhwana) [inline db/schema.rb:1418]: @lisasy you probably want to remove this schema change.

**Review by Ridhwana — COMMENTED** [review]
Looks good except for the schema change.

**Review by Zhao-Andy — APPROVED** [review]
LGTM, pending schema change removal!

**Review by Ridhwana — APPROVED** [review]

**Review by djuber — APPROVED** [review]
No idea if internationalization is ever on the horizon, but being able to change this label once, instead of in ten places would be a welcome side-effect of that effort.

**Review by jacobherrington — APPROVED** [review]
I think you got every instance where this needs to change! :+1:

**Pull request merged**